src/index.js

```diff
@@ -28,6 +29,19 @@ function init( {
 				if ( data ) {
 					popup.show( renderPreview( lang, data, isTouch ), target )
 				}
+
+				popup.lang = lang
+				popup.title = title
+				const expanded = root.querySelector( '.wikipediapreview.expanded.mobile' )
+
+				if ( expanded ) {
+					requestPageMedia( lang, title, mediaData => {
```

Comment:
Two cases depending on whether the preview text is long enough. If it's short enough and the gallery row fits then we `requestPageMedia` in index.js. Otherwise when user clicks continue reading CTA and expands the preview, we `requestPageMedia` in event.js.

Huei first noticed it [here](url) and that's how we had [ended up](https://github.com/wikimedia/wikipedia-preview/pull/21/commits/e05409353497ae51533f28562f38493256a61bf6) with `addMiniGalleryRow` in utils

Re-reading his comment again now I suppose we could make those few lines available through `events` when `popup` does `subscribe` if we want to make it more DRY